src/api/interceptors.ts

```diff
@@ -0,0 +1,76 @@
+import {
+  Fetcher,
+  FetcherErrorResponseInterceptor,
+  FetcherRequest,
+  FetcherRequestInterceptor,
+  FetcherResponse,
+  HTTP_METHODS,
+  HTTP_STATUS_CODES,
+} from '@distributedlab/fetcher'
+
+import { useAuthStore } from '@/store'
+import { i18n } from '@/localization'
+import { Bus } from '@/helpers'
+
+const REFRESH_TOKEN_ENDPOINT = '/integrations/nonce-auth-svc/refresh-token'
```

Comment:
? why resfresh and point here? If we have refresh method in store? Move refresh func to helpers or callers whatever and don't duplicate code